brave/brave-core
# Adblock custom filters

Fixes brave/brave-browser#403

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

- Visit brave://adblock
- Add `gravatar.com` to the custom Adblock rules
- Load `brianbondy.com` and verify that avatar icons don't load

## Reviewer Checklist:
